jni_gen/test/jackson_core_test/src/jackson_core_test.c

```diff
@@ -0,0 +1,2188 @@
+#include <stdint.h>
```

Comment:
Is this file generated? Then
* add a header similar to the generated file, and
* this file should have the copyright of the file it was generated from.